acceptance/testdata/builder/image/container/cnb/stack/buildpacks/example_stack/1.0.0/bin/build

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+echo "hello" > /file-to-snapshot
+echo "hello" > /bin/exe-to-snapshot
+chmod +x /bin/exe-to-snapshot
+
+# this should be ignored
+mkdir -p /cnb
+echo "hello" > /cnb/file-to-ignore
```

Comment:
Does it also make sense to check that the stack buildpack can't write to `/layers` and `/platform`?

Reply:
That could make sense, especially for `/platform`. It really isn't a test that they can't write somewhere, because they can. It is more a test of what we capture in the snapshot. Because they run as root, they'll be able to do anything to either directory and whatever changes will remain changed for the duration of the userspace buildpacks but will not be captured in the resulting snapshot.

Reply:
Ah, makes sense. I see now that `/layers` is tested in snapshot/kaniko_snapshot_unix_test.go...